doc/nrf/installation/install_ncs.rst

```diff
@@ -611,16 +619,15 @@ To install the |NCS| system-wide, complete the following steps:
 Installation with Toolchain Manager
 ***********************************
 
-.. note::
+Toolchain Manager is a SDK and toolchain installer available from `nRF Connect for Desktop`_, a cross-platform tool that provides different applications that simplify developing with the |NCS| and Nordic Semiconductor products.
```

Comment:
This still bothers me a bit as it has ... that... that...
How about something like: ... that provides different applications simplifying the development with...

Reply:
But this implies that just providing applications is simplifying the development :)